src/cmd/flux-wreckrun

```diff
@@ -89,12 +90,16 @@ local function alloc_tasks (f, wreck, lwj)
         end
     end
     for i, ntasks in pairs (counts) do
-        local key = "rank."..i..".cores"
-        lwj [key] = ntasks
+        local corelist = "0"
```

Comment:
Would it make sense here to keep the old rank.N.cores as an option for testing?

Reply:
we could I didn't want to add yet another option so in tests I just use --pre-launch-hook to do it. Easy to add an option though and that would be more straightforward.

Reply:
Oh as long as there is a way to drive it from tests.